packs/data/bestiary-effects.db/effect-crush-chitin.json

```diff
@@ -43,4 +43,4 @@
         "unidentified": false
     },
     "type": "effect"
-}
```

Comment:
?

Reply:
That automatically happens when I export the bestiary effects. Must be a line break at last line that slipped in while I redid the merge request for the opossums.